### PR TITLE
comments

### PR DESCRIPTION
**Pull request by sgaragagghu**

### CODE REVIEW

File: client/mining_common.h [hunk 1]
```diff
@@ -22,7 +22,7 @@ struct xdag_pool_task {
 };
 
 extern struct xdag_pool_task g_xdag_pool_task[2];
-extern uint64_t g_xdag_pool_task_index;
+extern uint64_t g_xdag_pool_task_index; // just a never initialized index, start value is not important
```
Comment (jonano614) [inline client/mining_common.h]: You can just fix it. Not necessary to describe the error.

Comment (sgaragagghu) [reply client/mining_common.h]: i don't think it is an error, it is just the way it works

Comment (jonano614) [reply client/mining_common.h]: This variable is always incremented. Also task index is writed to log.

Comment (sgaragagghu) [reply client/mining_common.h]: let me check

Comment (jonano614) [reply client/mining_common.h]: just set default value 
g_xdag_pool_task_index = 0
No need to write such long comments about nothing.

Comment (jonano614) [reply client/mining_common.h]: Take into account - global variables are always initialized with default value (0)

Comment (sgaragagghu) [reply client/mining_common.h]: didn't know, fixed.

**Pull request merged**